servers/physics_server.h

```diff
@@ -658,6 +658,26 @@ class PhysicsServer : public Object {
 	~PhysicsServer();
 };
 
+class PhysicsServerManager {
+	static Vector<String> physics_servers_classes;
```

Comment:
Just register function pointers with a name, using ClassDB names will restrict it from working using GDNative eventually if we want to provide external physics engines (such as PhysX or Havok)

Reply:
What do you advise me to use?